docs/source/user_guide/using_datasets.rst

```diff
@@ -4422,6 +4426,25 @@ FiftyOne currently supports
     3D meshes where possible, as it is the most compact, efficient, and
     web-friendly format for storing and transmitting 3D models.
 
+.. code-block:: python
+    :linenos:
+
+    import fiftyone as fo
+
+    scene = fo.Scene()
+
+    mesh1 = fo.GltfMesh("mesh1", "mesh.glb")
+    mesh1.rotation = fo.Euler(90, 0, 0, degrees=True)
+
+    mesh2 = fo.ObjMesh("mesh2", "mesh.obj")
+    mesh3 = fo.PlyMesh("mesh3", "mesh.ply")
+    mesh4 = fo.StlMesh("mesh4", "mesh.stl")
+    mesh5 = fo.FbxMesh("mesh5", "mesh.fbx")
+
+    scene.add(mesh1, mesh2, mesh3, mesh4, mesh5)
+
+    scene.write("/path/to/scene.fo3d")
```

Comment:
In the next iteration of our docs, we should also show examples for not just a flat scene, but something with some kind of hierarchy:
```
robotArmScene
-- base
---- arm
------ eoat

robot = fo.Scene()

base = fo.ObjMesh()
robot.add(base)

eoat = fo.ObjMesh()
base.add(eoat)
```

1st child of scene will be in world coordinates, whereas the subsequent children will be the coordinate system of their parents.